CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unversioned
 
+- Minor: Fixed double banphrase handling in the `safe_privmsg` function and `is_bad_message` function. (#1122)
```

Comment:
I don't think this actually needs a changelog entry because it doesn't change any behaviour.